.env.example

```diff
@@ -7,7 +8,13 @@ WALLARM_API_CA_VERIFY=True
 # Parameters of test account
 CLIENT_ID=...
 USER_TOKEN=...
-
-# Pytest variables (optional)
-# PYTEST_WORKERS=10
-# PYTEST_ARGS="--allure-features=Node"
+PYTEST_WORKERS=10
+PYTEST_ARGS="--allure-features=Node"
+ALLURE_ENVIRONMENT_ARCH = ...
+NODE_VERSION=...
+WEBHOOK_API_KEY=...
+WEBHOOK_UUID=...
+# not used when started localy
```

Comment:
if it's not required for a local run we do not need it here, cuz `.env` file is for local runs only

Reply:
updated